clientv3/example_watch_test.go

```diff
@@ -70,7 +70,7 @@ func ExampleWatcher_watchWithRange() {
 	}
 	defer cli.Close()
 
-	// watches within ['foo1', 'foo4'), in lexicographical order
```

Comment:
On Fri, May 20, 2016 at 5:21 PM, Anthony Romano <notifications@github.com>
wrote:

> It's [a,b). Cf.
> https://github.com/coreos/etcd/blob/master/mvcc/index.go#L157
>

I think we're looking at if the closing bracket should be round or square.